### PR TITLE
Adding information to the README.md file

### PR DESCRIPTION
**Pull request by amitCohen2**
Adding information about the app, team members, contribute, license and tools used in developing this application.

### CODE REVIEW

**Comment by Danielsio** [comment]
looks great. good job.

**Comment by MaayanMashhadi** [comment]
Looks great!
If the teammates like the logo you made, I suggest to add it.

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
-# beyond09-group01
\ No newline at end of file
+# MeetBalls
+
+Welcome to our new generation social network!
+'MeetBalls' is a web application that is supposed to help people to find partners,
```
Comment (yftacherzog) [inline README.md:4]:
```suggestion
'MeetBalls' is a web application that is aimed at helping people finding partners,
```
When saying `supposed to`, we often imply that it wasn't actually fulfilled.

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+* [Zvi Naor Cohen] (https://github.com/ZvikaNaorCohen)
+* [Daniel Sionov] (https://github.com/Danielsio)
+* [Ohad Segal] (https://github.com/OhadSeg)
+* [Amit Kaplansky] (https://github.com/amitKaplansky)
+* [Idan Zimilis] (https://github.com/IdanZimi)
+
+## Contribute
+
+In order to contribute, please refer to the contributing guide below.
+* [Contribute](https://github.com/redhat-beyond/MeetBalls/main/CONTRIBUTING.md/)
```
Comment (yftacherzog) [inline README.md:28]: Is there a reason this link does not use the same structure of the license file link below? I find the one for the license to make more sense.

**Review by yftacherzog — CHANGES_REQUESTED** [review]
Nicely done. Left a few comments.

File: README.md (hunk 1 above)
Comment (amitCohen2) [reply README.md]: Thanks, changed

File: README.md (hunk 2 above)
Comment (amitCohen2) [reply README.md]: Thank you for bringing this issue to my attention.
Changed.

**Comment by Danielsio** [comment]
Nicely done.

**Review by yftacherzog — APPROVED** [review]
@amitCohen2 Looks good! Let me know once you're done with making changes and I'll review it once more before merging it.

**Comment by amitCohen2** [comment]
> @amitCohen2 Looks good! Let me know once you're done with making changes and I'll review it once more before merging it.

Hi, I found some broken link and fixed it.
Done with changes:)

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+## Team members
+
+* [Amit Cohen] (https://github.com/amitCohen2)
+* [Maayan Mashhadi] (https://github.com/MaayanMashhadi)
+* [Zvi Naor Cohen] (https://github.com/ZvikaNaorCohen)
+* [Daniel Sionov] (https://github.com/Danielsio)
+* [Ohad Segal] (https://github.com/OhadSeg)
+* [Amit Kaplansky] (https://github.com/amitKaplansky)
+* [Idan Zimilis] (https://github.com/IdanZimi)
+
```
Comment (Yuval-Vino) [inline README.md:30]: seems to be a problem with the hyperlinks here ,  need to remove the space between the square brackets and the parentheses

Comment (yftacherzog) [reply README.md:30]: I didn't take into account that it should have not been displayed this way :sweat_smile: 
@amitCohen2 can you create an issue for fixing this? I think you can quickly create a PR to have it addressed.

Comment (amitCohen2) [reply README.md:30]: New PR for fixing this annoying issue added

**Pull request merged**